base_location_geonames_import/README.rst

```diff
@@ -1,87 +1,113 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
-   :alt: License: AGPL-3
-
 =============================
 Base Location Geonames Import
 =============================
 
-This module adds a wizard to import cities and/or better zip entries from
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
+    :target: https://github.com/OCA/partner-contact/tree/12.0/base_location_geonames_import
+    :alt: OCA/partner-contact
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/partner-contact-12-0/partner-contact-12-0-base_location_geonames_import
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/134/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a wizard to import cities and/or city zip entries from
 `Geonames <http://www.geonames.org/>`_ database.
 
+**Table of contents**
+
+.. contents::
+   :local:
+
 Installation
 ============
 
-To install this module, you need the Python library 'requests'.
+To install this module, you need the Python library 'requests'::
+
+    pip install requests
+
 
 Configuration
 =============
 
-To access the menu to import better zip entries from Geonames,
+To access the menu to import city zip entries from Geonames
 you must add yourself to the groups *Technical features* and *Sales manager*.
```

Comment:
This has changed in this version

Reply:
Its ```Administration / Settings``` now, right?

Reply:
I'm not sure. Check Contacts > Configuration group.